recipes/openvino/all/conanfile.py

```diff
@@ -0,0 +1,402 @@
+from conan import ConanFile, conan_version
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd, cross_building
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, rmdir
+import os
+
+required_conan_version = ">=1.60.0 <2.0 || >=2.0.8"
+
+class OpenvinoConan(ConanFile):
+    name = "openvino"
+
+    # Optional metadata
+    license = "Apache-2.0"
+    homepage = "https://github.com/openvinotoolkit/openvino"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Open Visual Inference And Optimization toolkit for AI inference"
+    topics = ("nlp", "natural-language-processing", "ai", "computer-vision", "deep-learning", "transformers", "inference",
+              "speech-recognition", "yolo", "performance-boost", "diffusion-models", "recommendation-system", "stable-diffusion",
+              "generative-ai", "llm-inference", "optimize-ai", "deploy-ai")
+    package_id_non_embed_mode = "patch_mode"
+    package_type = "library"
+    short_paths = True
+    no_copy_source = True
+
+    # Binary configuration
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        # HW plugins
+        "enable_cpu": [True, False],
+        "enable_gpu": [True, False],
+        # SW plugins
+        "enable_auto": [True, False],
+        "enable_hetero": [True, False],
+        "enable_auto_batch": [True, False],
+        # Frontends
+        "enable_ir_frontend": [True, False],
+        "enable_onnx_frontend": [True, False],
+        "enable_tf_frontend": [True, False],
+        "enable_tf_lite_frontend": [True, False],
+        "enable_paddle_frontend": [True, False],
+        "enable_pytorch_frontend": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        # HW plugins
+        "enable_cpu": True,
+        "enable_gpu": True,
+        # SW plugins
+        "enable_auto": True,
+        "enable_hetero": True,
+        "enable_auto_batch": True,
+        # Frontends
+        "enable_ir_frontend": True,
+        "enable_onnx_frontend": True,
+        "enable_tf_frontend": True,
+        "enable_tf_lite_frontend": True,
+        "enable_paddle_frontend": True,
+        "enable_pytorch_frontend": True
+    }
+    options_description = {
```

Comment:
This feature is not documented as far as I can tell so it's probably too early to introduce it in CCI.
I would also say these are not very descriptive and do not add much so I think it fine to remove them.

Reply:
removed